io/include/pcl/io/real_sense/real_sense_device_manager.h

```diff
@@ -119,21 +123,19 @@ namespace pcl
           void
           populateDeviceList ();
 
-          std::shared_ptr<PXCSession> session_;
-          std::shared_ptr<PXCCaptureManager> capture_manager_;
+          PXCSessionPtr session_;
+          PXCCaptureManagerPtr capture_manager_;
 
           std::vector<DeviceInfo> device_list_;
 
           static std::mutex mutex_;
-
       };
 
+
       class PCL_EXPORTS RealSenseDevice : boost::noncopyable
       {
-
         public:
-
-          using Ptr = std::shared_ptr<RealSenseDevice>;
+          using RealSenseDevicePtr = pcl::shared_ptr<RealSenseDevice>;
```

Comment:
Revert this please.

Reply:
`Ptr`->`RealSenseDevicePtr` or `std`->`pcl::`?

Reply:
If it's because it's breaking API, then I need to revert most of the changes here. Consider that I'm ready to push #3750 so the change can be merged sequentially, hence unless one is unlucky to pull master in the timeframe where one commit is merged and the other is not...I mean...anyways master isn't supposed to be stable, right?

Reply:
What needs to be changed might actually be https://github.com/PointCloudLibrary/pcl/blob/655a422c42ef5518feab2285e1d53181a5a339fe/io/include/pcl/io/real_sense/real_sense_device_manager.h#L148